src/main/java/org/mskcc/cmo/messaging/impl/MessageConsumerImpl.java

```diff
@@ -0,0 +1,17 @@
+package org.mskcc.cmo.messaging.impl;
```

Comment:
Do we need this file?  Is it default behavior?

Reply:
Have we decided what we want the default behavior to be?

Reply:
I don't think we should have any default behavior.  I'm going to let this in so the PR can be merged while you are away.